lib/flutter_periculum.dart

```diff
@@ -68,4 +69,46 @@ class FlutterPericulum {
 
     return jsonEncode(myresponse).toString();
   }
+
+  static Future<dynamic> statementAnalytics({required String token}) async {
```

Comment:
there should be no dynamics in this file. We should have models for everything. In this call, make it return an Analytics model.